python/.gitignore

```diff
@@ -0,0 +1,4 @@
+docs/_build
+docs/build
+*.so
+*.pyc
```

Comment:
What about adding all of these to the root .gitignore?

Reply:
Sure, I can do that.

Reply:
Done.